server/index.js

```diff
@@ -1,22 +1,8 @@
-require('newrelic');
+const app = require('./routes');
 
-const express = require('express');
-const path = require('path');
-const morgan = require('morgan');
-const { createProxyMiddleware } = require('http-proxy-middleware');
+require('dotenv').config();
 
-const app = express();
-
-app.use(morgan('dev'));
-app.use(express.static(path.join(__dirname, '../public')));
-
-app.use(
-  '/shop/:productId/:styleId',
-  createProxyMiddleware({ target: 'http://localhost:3004', changeOrigin: true })
-);
-
-// port 4000 for the proxy server
-const port = 4000;
+const port = process.env.PORT || 4000;
```

Comment:
Looks like you're moving code to separate file to separate concerns?
Looks good.

Reply:
Yes, my apologies, there should've been an annotation for this one. I've essentially reduxed the index file so it only handles the App listen method, and the routes live in it's own file.